tsconfig.json

```diff
@@ -6,14 +6,17 @@
     "sourceMap": true,
     "noImplicitAny": true,
     "noUnusedLocals": true,
+    "resolveJsonModule": true,
     // TODO(rsgowman): enable `"strict": true,` and remove explicit setting of: noImplicitAny, noImplicitThis, alwaysStrict, strictBindCallApply, strictNullChecks, strictFunctionTypes, strictPropertyInitialization.
     "noImplicitThis": true,
     "alwaysStrict": true,
     "strictBindCallApply": true,
     "strictNullChecks": true,
     "strictFunctionTypes": true,
     //"strictPropertyInitialization": true,
-    "lib": ["es2018"],
+    "lib": [
+      "es2018",
+      "dom"],
```

Comment:
closing bracket should be on a separate line.

Reply:
Fixed!